### ISSUE TITLE
Where is the testing_emonet_split.csv?

### ISSUE THREAD

**Issue by Cryst4L9527** [radekd91/emoca#80]
I'm trying to reproduce the training and testing of emotion recognition, but it seems the testing_emonet_split.csv is not provided.

**Comment by Cryst4L9527** [comment]
I’ve tried the training and testing of emotion recognition with the pretrained model released(EMOCA/detail/checkpoints/deca-epoch=03-val_loss/dataloader_idx_0=9.44489288.ckpt) and the training.csv and validation.csv provided by the EmoNet, but the results is much lower than the report in the paper. I wonder if a clearer code about this metric can be released to reproduce the results in the paper?
